### ISSUE TITLE
chore: Make packages/medusa/src/services/product-variant.js pass linting

### ISSUE THREAD

**Issue by srindom** [medusajs/medusa#518]
In an effort to improve our code quality we want to have make sure that all files in the repo pass linting.

Please see this guide for a step-by-step guide for how to complete this task: https://www.notion.so/medusajs/Lint-challenge-checklist-b4efc761cb36499abf4726cad0dd02e4

🚀

**Comment by Mihir50** [comment]
May I try this?

**Comment by Mihir50** [comment]
When I run yarn lint, i get this:

```
error  Delete `␍`  prettier/prettier
✖ 5289 problems (5289 errors, 0 warnings)
```
On stackoverflow, they say to change '**endOfLine**' to '**auto**' in '**prettierrc**' file. What should I do?

**Comment by olivermrbl** [comment]
Many thanks @Mihir50 ! Would love to send out some swag for you to show our appreciation. Reach out to me on our Discord or via email, if you are interested ❤️ 

Closed #693

**Comment by Mihir50** [comment]
> Many thanks @Mihir50 ! Would love to send out some swag for you to show our appreciation. Reach out to me on our Discord or via email, if you are interested ❤️
> 
> Closed #693

Sure. Thank you 👍
